packages/hasura-auth-js/src/hasura-auth-client.ts

```diff
@@ -283,7 +283,7 @@ export class HasuraAuthClient {
       interpreter.send({ type: 'SIGNOUT', all: params?.all })
       interpreter.onTransition((state) => {
         if (state.matches({ authentication: { signedOut: 'success' } })) resolve({ error: null })
```

Comment:
to improve readability, I'd suggest that we should use curly braces around these statements